recipes/msgpack-c/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "msgpack-c" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "eb20b4bf15f20bad149ec82fffac74f16de2a8a797e321a3f8189d63263a511e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: cpp-{{ version }}.tar.gz
+  url: https://github.com/msgpack/{{ name }}/archive/cpp-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - toolchain
+    - boost
+    - zlib
+
+test:
+  commands:
+    - test -f $PREFIX/include/msgpack.h                         # [unix]
+    - test -f $PREFIX/include/msgpack.hpp                       # [unix]
+    - test -f $PREFIX/lib/libmsgpackc.a                         # [unix]
+    - test -f $PREFIX/lib/libmsgpackc.so                        # [linux]
+    - test -f $PREFIX/lib/libmsgpackc.dylib                     # [osx]
+    - if not exist %LIBRARY_INC%\\msgpack.h exit 1              # [win]
+    - if not exist %LIBRARY_INC%\\msgpack.hpp exit 1            # [win]
+    - if not exist %LIBRARY_LIB%\\msgpackc.lib exit 1           # [win]
```

Comment:
This is the static library, can you rename this to `msgpackc_static.lib` and the import library `msgpackc_import.lib` to `mspackc.lib`?